Feat: Eventing support in Service Catalog (#8075)
Kong/docs.konghq.com: app/konnect/service-catalog/integrations/index.md

---

Integrations are applications - either {{site.konnect_short_name}}-internal applications or external 

When enabled, an integration gives you and your teams the ability to:
* Discover new resources
* Create new services based on existing resources

> * View key events associated with the integration directly in the event tab in {{site.konnect_short_name}}. 

Reviewer: ```suggestion
* View key events associated with the integration directly in the event tab in {{site.konnect_short_name}}
```
The other bullet points don't have `.`

## Integrations

{% include_cached service-catalog-integrations.html %}

## Events

> Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_short_name}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.

Reviewer: ```suggestion
Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_short_name}} UI. 
From this tab, you can search or filter through events by keywords in the event title and filter by time frame.
```
"Here" seems ambiguous.

> ## Events
> 
> Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_short_name}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.

Reviewer: ```suggestion
```
This doesn't belong here. As this page grows, it will have many more tiles. This info will get obscured, and will seem even more out of place. Can you please move it to the service catalog index page?

Author: Will do in post

---

Integrations are applications - either {{site.konnect_short_name}}-internal applications or external 
third-party applications - that you can enable or disable within {{site.service_catalog_name}}.

When enabled, an integration gives you and your teams the ability to:
* Discover new resources
* Create new services based on existing resources

> * View key events associated with the integration directly in the event tab in {{site.konnect_saas}}. 

Reviewer: ```suggestion
* View key events associated with the integration directly in the event tab in {{site.konnect_short_name}}. 
```

## Integrations

{% include_cached service-catalog-integrations.html %}

## Events

> Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the Events tab in the {{site.konnect_saas}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.

Author: ```suggestion
Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_saas}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.
```

> Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_saas}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.

Reviewer: ```suggestion
Eventing allows you to view key events associated with your {{site.service_catalog_name}} integrations directly from the **Events** tab in the {{site.konnect_short_name}} UI. From here you can search or filter through events by keywords in the event title and filter by time frame.
```
